internal/cache/content_cache_data_test.go

```diff
@@ -25,7 +25,7 @@ func TestContentCacheForData(t *testing.T) {
 		Storage:    cacheStorage,
 		HMACSecret: []byte{1, 2, 3, 4},
 		Sweep: cache.SweepSettings{
-			MaxSizeBytes: 100,
+			MaxSizeBytes: 150,
```

Comment:
yes, good question. I should put a comment there

each protected blob (hashed blob) takes up about 35bytes while the payload is only 3 bytes, which was not the case originally. the original code was tracking the payload bytes but I made a change to track real bytes on disk (protected bytes) in cache. reason for this change was to correctly track how much space we are using to avoid over the limit

so, had to adjust the unit-test accordingly